feature/feature.go

```diff
@@ -74,6 +74,9 @@ const (
 
 	// ResourceCleanup will start gc controller to clean up resources in cluster ns after cluster is deleted.
 	ResourceCleanup featuregate.Feature = "ResourceCleanup"
+
+	// SwitchHub allows user to configure multiple bootstrapkubeconfig via Klusterlet and let agent to decide which one to use
+	SwitchHub featuregate.Feature = "SwitchHub"
```

Comment:
MultipleHubs might be a better name. Thought @elgnay ?

Reply:
Agree, the `MultipleHubs` is better, fixed.